ldap-connector/config/variables.yaml

```diff
@@ -6,4 +6,7 @@
 # '<project>/Config/_<environment>/variables.yaml
```

Comment:
we need to add more meta infos by adding description to all of these variables. but we can make that at the end.